main.go

```diff
@@ -482,6 +498,9 @@ func Process(s *State) {
 	processorGroup.Wait()
 	close(resultChan)
 	printerGroup.Wait()
+	if s.OutputFileName != "" {
```

Comment:
yep, I agree! Would you like me to push down the changes (s.outputFileName and -cn flag)?